iocore/aio/AIO.cc

```diff
@@ -475,9 +475,9 @@ aio_thread_main(void *arg)
         SCOPED_MUTEX_LOCK(lock, op->mutex, thr_info->mutex->thread_holding);
         op->handleEvent(EVENT_NONE, nullptr);
       } else if (op->thread == AIO_CALLBACK_THREAD_ANY) {
-        eventProcessor.schedule_imm_signal(op);
```

Comment:
So how is the AIO thread signaling the event thread to wake up now?

Reply:
yep, fast signal was removed during the big eventloop change PR #2541, kept putting it off but. This is also related to #6103